crates/biome_string_case/src/lib.rs

```diff
@@ -447,6 +461,10 @@ impl StrExtension for std::ffi::OsStr {
             Cow::Borrowed(self)
         }
     }
+
+    fn to_lowercase_cow(&self) -> Cow<Self> {
+        panic!("OsStr is not supported")
+    }
```

Comment:
I am not sure what todo here. OsStr does not support to_lowercase. Implementing it manually is hard and we don't really need it.

Is leaving as is fine? Do we need separate trait?

Reply:
I suppose a separate trait would be safest.

Reply:
I split it into two separate traits, shame [trait-alias](https://doc.rust-lang.org/unstable-book/language-features/trait-alias.html) are not stable so only one trait could be imported for both. We could use [trait-set macro](https://crates.io/crates/trait-set/0.3.0) but I imagine introducing new crate for this would be overkill.